aria-practices.html

```diff
@@ -6338,6 +6338,115 @@ <h3>Indicating sort order with <code>aria-sort</code></h3>
 
   </section>
 
+  <section id="aria-level">
+    <h2>Describing Hierarchical Structure with <code>aria-level</code></h2>
+    <p>When elements have a hierarchical relationship, such as section headings or data in tree structures, <code>aria-level</code> is used to communicate the hierarchy between elements to assistive technologies. The value of <code>aria-level</code> is numeric, with <code>1</code> indicating the top level of the structure. The number increases for each level of nesting.</p>
+    <p>Do not use <code>aria-level</code> when the DOM structure already accurately represents the elements' hierarchical relationships. For example, a nested <code>ul</code> list structure does not need explicit <code>aria-level</code>.</p>
+    <p>The <code>aria-level</code> attribute can be used on the following roles:</p>
+    <ul>
+      <li><code>heading</code></li>
+      <li><code>listitem</code></li>
+      <li><code>treeitem</code></li>
+      <li><code>row</code></li>
+    </ul>
+    <section id="aria-level_heading_role">
+      <h3><code>heading</code> Role</h3>
```

Comment:
Make headings more verbose/redundant so they can stand alone better. Like "Using aria-level for the heading Role".